samcli/lib/cookiecutter/interactive_flow.py

```diff
@@ -76,17 +82,16 @@ def run(self, context: Dict) -> Dict:
                  associated to the key of the corresponding question
         """
         context = context.copy()
+        question_handlers = {
+            QuestionKind.info: InteractiveFlow._echo,
+            QuestionKind.choice: InteractiveFlow._ask_a_multiple_choice_question,
+            QuestionKind.confirm: InteractiveFlow._ask_a_yes_no_question,
+            QuestionKind.default: InteractiveFlow._ask_a_question,
+        }
```

Comment:
nit:

`question_handlers` can be put in out scope. And can we add one more layer of safety in case in the future we have more kinds of questions with this unit test?

```py
for kind in QuestionKind:
    self.assertIn(kind, InteractiveFlow.question_handlers)
```

Reply:
will add the unit test. Not sure about the scope of `question_handlers` though, it is only required here

Reply:
we can also leave it in there, then we just need to unit test the whole method.